__tests__/app.test.js

```diff
@@ -337,3 +347,28 @@ describe("PATCH /api/reviews/review_id", () => {
       });
   });
 });
+describe("/api/users", () => {
+  test("GET - STATUS: 200, responds with an array of the newly inserted user objects", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { user } }) => {
+        expect(users).toHaveLength(4);
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("404: responds with correct error status when invalid path has been used", () => {

```

Comment:
I know it feels like there isn't much to test for here but that's fine, I think you might have already created the error handling to cover this, I dont think you need to add it here too :)